services/terraformer/templates/oci.tpl

```diff
@@ -205,11 +218,12 @@ resource "oci_core_volume" "{{ $node.Name }}_volume" {
 }
 
 resource "oci_core_volume_attachment" "{{ $node.Name }}_volume_att" {
-  provider         = oci.k8s_nodepool_{{ $nodepool.Region }}
+  provider        = oci.k8s_nodepool_{{ $nodepool.Region }}
   attachment_type = "paravirtualized"
   instance_id     = oci_core_instance.{{ $node.Name }}.id
   volume_id       = oci_core_volume.{{ $node.Name }}_volume.id
   display_name    = "{{ $node.Name }}-volume-att"
+  device          = "/dev/oracleoci/${var.storage_disk_name}"
```

Comment:
hmm this is how it's done! good stuff.
it'd be awesome if OCI allowed defining truly custom device names but hey, we can't have it all..

Reply:
Yeah, similar to other providers but just enough different to cause me headache :)